AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -26,6 +26,7 @@ quinhillyer.com,johnbiver.com,jasonmattera.com,ipatriot.com,hooligancharters.com
 ##.widget_blog_subscription
 ~curvegames.com,~news.un.org,~norwichzen.org.uk,~dereferer.me###mc_embed_signup
 !
+crm.ru##.subscribe-form
```

Comment:
@spiri-leo
Please add the domain to the existing rule

Reply:
Fixed.